test/e2e/utils/fixture/fixture.go

```diff
@@ -0,0 +1,96 @@
+package fixture
+
+import (
+	"context"
+	"fmt"
+	topologyclientset "github.com/k8stopologyawareschedwg/noderesourcetopology-api/pkg/generated/clientset/versioned"
+	. "github.com/onsi/gomega"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"k8s.io/client-go/kubernetes"
+
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/client/config"
+)
+
+var F Fixture
+var initialized bool
+
+type Fixture struct {
+	Ctx     context.Context
+	Cli     client.Client
+	K8SCli  *kubernetes.Clientset
+	TopoCli *topologyclientset.Clientset
+	// TestingNS randomly generated for each test and gets deleted when the test ends
+	TestingNS *corev1.Namespace
+}
+
+func init() {
+	// important so we keep the same context across the suite run
+	if initialized {
+		return
+	}
+	F.Ctx = context.Background()
+	Expect(initK8SClient()).ToNot(HaveOccurred())
+	Expect(initClient()).ToNot(HaveOccurred())
+	Expect(initTopologyClient()).ToNot(HaveOccurred())
```

Comment:
IIRC `Expect` outside a `It`(/`Describe`...) block are not legal

Reply:
I saw the k8s e2e framework using similar approach:
https://github.com/openshift-kni/numaresources-operator/blob/ea32cc11d13479c6a4e5091f7c080040bd67d4b1/vendor/k8s.io/kubernetes/test/e2e/framework/log.go#L61 
so  I thought it's OK